js/spotify.js

```diff
@@ -1,12 +1,43 @@
-// API Docs at: 
+// API Docs at:
 // https://developer.spotify.com/web-api/search-item/
+$(document).ready(function () {
+  $('form').on('submit', function (e) {
+    e.preventDefault();
+  var keyword = $('#search-keyword').val();
+  searchByArtist();
+  })
+
+
+
 
 
 function searchByArtist(keyword) {
-  var url = 'https://api.spotify.com/v1/search?q='+keyword+'&type=artist';
-}
+     var url = 'https://api.spotify.com/v1/search?q='+keyword+'&type=artist';
+     var request = $.ajax(url);
+
+     request.done(function (response) {
+       data = response;
```

Comment:
Make sure to declare your variables here. 
```
var data = response;
var artist = response.artist;
```

You only want to stick with convention in using the $ before variables if the variable itself is a jquery object. In the case of artist, it is a value in data so the $ isn't needed.